src/components/TxBuilderAttributes.js

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState } from 'react';
-import lyraApi from "@stellar/lyra-api";
+import { isConnected, getPublicKey } from "@stellar/lyra-api";
```

Comment:
ah, so I changed this to `import lyraApi from "@stellar/lyra-api";` so when you call things like `isConnected`, in the code, it's clear it's a Lyra method

I'm fine with either, honestly. Looks like we destructure methods from the other imports, so maybe it's best to do it this way

Reply:
Yeah I think either pattern is fine to use